packages/tabs/test/tabs-overflow.test.ts

```diff
@@ -0,0 +1,161 @@
+/*
+Copyright 2022 Adobe. All rights reserved.
+This file is licensed to you under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License. You may obtain a copy
+of the License at http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software distributed under
+the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR REPRESENTATIONS
+OF ANY KIND, either express or implied. See the License for the specific language
+governing permissions and limitations under the License.
+*/
+import { elementUpdated, expect, fixture } from '@open-wc/testing';
+import { html, nothing } from '@spectrum-web-components/base';
+import { repeat } from 'lit/directives/repeat.js';
+
+import { TabsOverflow } from '../src/TabsOverflow';
+import { Tab } from '../src/Tab';
+import { ActionButton } from '@spectrum-web-components/action-button';
+import '@spectrum-web-components/theme/sp-theme.js';
+import '@spectrum-web-components/theme/scale-medium.js';
+import '@spectrum-web-components/theme/theme-light.js';
+import '../sp-tabs-overflow.js';
+import '../sp-tabs.js';
+import '../sp-tab.js';
+import '../sp-tab-panel.js';
```

Comment:
Try updating these to `@spectrum-web-components/tabs/...` rather than `../` the relative path doesn't register the dev vs prod build separations and might be messing up the code coverage.